flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackendTest.java

```diff
@@ -250,7 +251,10 @@ public void testCorrectMergeOperatorSet() throws IOException {
 				enableIncrementalCheckpointing,
 				TestLocalRecoveryConfig.disabled(),
 				RocksDBStateBackend.PriorityQueueStateType.HEAP,
-				TtlTimeProvider.DEFAULT);
+				TtlTimeProvider.DEFAULT,
+				new RocksDBNativeMetricOptions(),
+				Optional.empty()
```

Comment:
If this argument an optional purely to simplify tests? If so, please just use an `UnregisteredOperatorMetricGroup` instead.